tailwind.config.js

```diff
@@ -650,6 +641,9 @@ module.exports = {
         '1/2': '50%',
         18: '4.5rem'
       },
+      zIndex: {
```

Comment:
Please revert removing other values of `zIndex` because they are used in lots of components